trino/client.py

```diff
@@ -559,6 +562,10 @@ def is_finished(self):
         # type: () -> bool
         return self._finished
 
+    def is_cancelled(self):
```

Comment:
Can you make this a property?

Reply:
Yeah, sure. And `is_finished` as wel?

Reply:
In general we should align on using properties since that is what has been done for most attributes, but it's optional since you didn't touch it haha

Reply:
I create `finished` property and now `is_finished()` marked as deprecated.